packages/renderer/src/lib/ui/LinearProgress.svelte

```diff
@@ -0,0 +1,64 @@
+<style>
+.pure-material-progress-linear {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: none;
+    height: 0.10em;
+    color: var(--pf-global--primary-color--100);
+    font-size: 16px;
+}
+
+.pure-material-progress-linear::-webkit-progress-bar {
+    background-color: transparent;
+}
+
+/* Determinate */
+.pure-material-progress-linear::-webkit-progress-value {
+    background-color: currentColor;
+    transition: all 0.2s;
+}
+
+.pure-material-progress-linear::-moz-progress-bar {
+    background-color: currentColor;
+    transition: all 0.2s;
+}
```

Comment:
note sure about the usage of the mozilla properties, webkit and other properties
AFAIK in electron we have only one browser so we don't need to handle all browsers